grails-app/domain/org/pih/warehouse/product/ProductSupplier.groovy

```diff
@@ -160,12 +162,14 @@ class ProductSupplier implements Serializable, Comparable<ProductSupplier> {
             productId: product.id,
             productName: product.name,
             code: code,
-            supplierName: supplier?.name,
+            supplierName: supplier?.name + (supplier?.code ? " (${supplier.code})" : ""),
             supplierCode: supplierCode,
             productSupplierPreferences: productSupplierPreferences.collect { it.toJson() },
             packageSize: defaultProductPackage ? "${defaultProductPackage?.uom?.code}/${defaultProductPackage?.quantity}" : null,
-            packagePrice: defaultProductPackage?.productPrice?.price ?: 0.0,
-            unitPrice: defaultProductPackage?.productPrice ? defaultProductPackage?.productPrice?.price / defaultProductPackage?.quantity : 0.0,
+            packagePrice: defaultProductPackage?.productPrice?.price?.setScale(2, RoundingMode.HALF_UP) ?: 0.0,
```

Comment:
I want to think about this a little more. We've used this pattern twice now so it probably deserves it's own transient property getter or a utility.

Reply:
Let's move it to a separate ticket or spike if needed. For now, I am merging this to the feature branch.